### PR TITLE
Misc updates to ur_description

### PR DESCRIPTION
**Pull request by gavanderhoorn**
This is a set of misc updates to the `ur_description` package, on-top of the updates @ipa-led contributed in #371.

Mostly layout and formatting, but f7174e2a changes the filenames of the main `.xacro` files to be more in-line with other ROS-Industrial support packages:

 - `ur_macro.xacro`: this file contains the base xacro macro. It is only a macro, it cannot be loaded directly
 - `ur.xacro`: this is a top-level xacro. Passing this file through `xacro` results in a legal, flattened `.urdf`

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
It's a big PR in the sense it contains quite some commits, but each commit is rather small, so I believe this should still be reviewable.

File: ur_description/urdf/ur_macro.xacro [hunk 1]
```diff
@@ -1,25 +1,87 @@
...
+    offets, etc) do not correspond to any particular robot. They are defaults
+    only. There WILL be non-zero offsets between the Forward Kinematics results
+    in TF (ie: robot_state_publisher) and the values reported by the Teach
+    Pendant.
+
+    For accurate (and robot-specific) transforms, the 'kinematics_parameters_file'
+    parameter MUST point to a .yaml file containing the appropriate values for
+    the targetted robot.
+
+    If using the UniversalRobots/Universal_Robots_ROS_Driver, follow the steps
```
Comment (fmauch) [inline ur_description/urdf/ur_macro.xacro:27]: We could add a link to the README here?

File: ur_description/urdf/ur_macro.xacro [hunk 2]
```diff
@@ -1,25 +1,87 @@
...
+     - Shaun Edwards
+     - Nadia Hammoudeh Garcia
+     - Dave Hershberger
+     - G. vd. Hoorn
+     - Philip Long
+     - Dave Coleman
+     - Miguel Prada
+     - Marcel Schnirring
+     - Felix von Drigalski
+     - Felix Mauch
```
Comment (fmauch) [inline ur_description/urdf/ur_macro.xacro]:
```suggestion
     - Felix Exner
```

File: ur_description/urdf/ur_macro.xacro [hunk 3]
```diff
@@ -1,25 +1,87 @@
...
-    <xacro:read_model_data joints_limits_parameters_file="${joints_limits_parameters_file}" 
-                    kinematics_parameters_file="${kinematics_parameters_file}"
-                    physical_parameters_file="${physical_parameters_file}"
-                    visual_parameters_file="${visual_parameters_file}"/>
+    <!-- This is for compatibility reasons, as transformation between base and
+         base_link was always 180 degree in the ur_description.
 
-    <!-- This is for compatibility reasons, as transformation between base and base_link was always 180 degree in the ur_description -->
+         NOTE: this will be removed in a future revision of this file
+     -->
```
Comment (fmauch) [inline ur_description/urdf/ur_macro.xacro]: This was for example discussed inside https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/issues/78

But I guess this should be addressed in a different PR?

**Review by fmauch — APPROVED** [review]
Most of all just minor comments. It would be nice, however, if my name would be updated :-)

File: ur_description/urdf/ur_macro.xacro (hunk 2 above)
Comment (gavanderhoorn) [reply ur_description/urdf/ur_macro.xacro]: The commit still said `Mauch`, that's why it ended up like that in this list.

File: ur_description/urdf/ur_macro.xacro (hunk 3 above)
Comment (gavanderhoorn) [reply ur_description/urdf/ur_macro.xacro]: Yes, I did not want to do that in this PR.

File: ur_bringup/launch/ur10_bringup.launch [hunk 4]
```diff
@@ -19,7 +19,7 @@
   <arg name="visual_params" default="$(find ur_description)/config/ur10/visual_parameters.yaml"/>
 
   <!-- robot model -->
-  <include file="$(find ur_description)/launch/ur10_upload.launch">
+  <include file="$(find ur_description)/launch/load_ur10.launch">
```
Comment (gavanderhoorn) [inline ur_bringup/launch/ur10_bringup.launch:22]: `ur_bringup` will be removed, but in a later PR.

So I had to fix these lines.

File: ur_description/urdf/ur_macro.xacro (hunk 1 above)
Comment (gavanderhoorn) [reply ur_description/urdf/ur_macro.xacro:27]: We could, but I feel it's rather clear where to find it (ie: the readme of the repository I just mentioned).

If you google for `UniversalRobots/Universal_Robots_ROS_Driver` it's literally the first result.

**Review by dhled — APPROVED** [review]
Thank you for this addings.

**Comment by gavanderhoorn** [comment]
Seeing as Travis is now happy and we have two approving reviews, I'm going to merge this.

Thanks @fmauch and @ipa-led for the reviews 👍
